Misc/NEWS.d/next/Core_and_Builtins/2025-07-19-10-35-31.gh-issue-74185.7hPCA5.rst

```diff
@@ -0,0 +1,4 @@
+The :meth:`~object.__repr__` of :class:`ImportError` and :class:`ModuleNotFoundError`
+now shows "name" and "path" as ``name=<name>`` and ``path=<path>`` if they were given
+as keyword arguments at construction time.
+(Contributed by Serhiy Storchaka, Oleg Iarygin, and Yoav Nir in :gh:`74185`.)
```

Comment:
```suggestion
Patch by Serhiy Storchaka, Oleg Iarygin, and Yoav Nir.
```

For blurb entries, there's already a "gh-XXXX" as a prefix so no need to indicate them here as well. Usually credits are sufficient.

Reply:
I would still prefer using "Patch by " because that's what most entries do: https://docs.python.org/3.15/whatsnew/changelog.html. Or at least remove the parentheses.